LayoutTests/imported/w3c/web-platform-tests/trusted-types/support/navigation-support.js

```diff
@@ -59,7 +59,9 @@ async function setLocationToJavaScriptURL(defaultpolicy) {
         // observed violations, otherwise we could miss the corresponding
         // pre-navigation check CSP violation.
         if (window.requestIdleCallback) {
-          await new Promise(resolve => requestIdleCallback(resolve));
+            await new Promise(resolve => requestIdleCallback(resolve));
+        } else {
+          await new Promise(resolve => requestAnimationFrame(_ => requestAnimationFrame(resolve)));
```

Comment:
Inconsistent indentation here
```suggestion
          await new Promise(resolve => requestIdleCallback(resolve));
        } else {
          await new Promise(resolve => requestAnimationFrame(_ => requestAnimationFrame(resolve)));
```